src/Microsoft.Azure.WebJobs.Host/Indexers/FunctionIndexer.cs

```diff
@@ -206,6 +206,13 @@ private bool HasJobAttribute(CustomAttributeData attributeData)
             if (triggerBinding != null)
             {
                 bindingDataContract = triggerBinding.BindingDataContract;
+                
+                // See if a regular binding can handle it. 
```

Comment:
>// See if a regular binding can handle it.  [](start = 16, length = 43)

This is part of the support to let Triggers leverage Converters with a BindToTrigger<> rule.